packages/dashboard/src/components/side-panel/sub-components/chart-settings.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { ExpandableSection } from '@cloudscape-design/components';
+
+const ChartSettings = () => {
+  return (
+    <ExpandableSection headerText="Chart settings" defaultExpanded>
```

Comment:
In the sub-components I'd have a `const [expanded, setExpanded] = useState<boolean>(true);` instead of using defaultExpanded. 

Then you can pass something to toggle it in through the onClick and have that working right away. Removing defaultExpanded will get rid of another prop in the future.

Reply:
We don't need to control state of expandable section. ExpandableSection handles that by itself.